src/components/Composer/index.tsx

```diff
@@ -334,11 +258,11 @@ function Composer({
      * divide by line height to get the total number of rows for the textarea.
      */
     const updateNumberOfLines = useCallback(() => {
-        if (textInput.current === null) {
+        if (!textInput.current) {
             return;
         }
         // we reset the height to 0 to get the correct scrollHeight
-        textInput.current.style.height = 0;
+        textInput.current.style.height = '0';
```

Comment:
Is this on purpose?

Reply:
Yes, number is not assignable to this param